src/util_c_variables.nss

```diff
@@ -89,3 +89,32 @@ string ObjectToDatabaseTag(object oSource, object oDatabase, string sVarName,
 {
     return sTag;
 }
+
+// -----------------------------------------------------------------------------
+//                          Debugging Messages
+// -----------------------------------------------------------------------------
+// This function is called when a debug message needs to be sent due to errors
+//  int the system.  Any debug system may be used through this conduit.  The
+//  sample code uses sm-util's util_i_debug.nss.
+// -----------------------------------------------------------------------------
+
+// These are generic constants and can be modified or deleted at anytime to
```

Comment:
If these are being referenced in `util_i_variables`, they can't be deleted.

Did you have a particular use case that made you want to do it this way instead of just using `util_i_debug` directly?